src/openforms/config/templates/emails/confirmation_email_content.html

```diff
@@ -0,0 +1,13 @@
+Geachte heer/mevrouw,<br>
```

Comment:
It did seem to work with `{% blocktrans %}` or would this not be correct?

![Screenshot 2021-11-17 at 11 04 06](https://user-images.githubusercontent.com/60747362/142179907-2443febd-31cf-494a-bfa4-a5c27f6b2c51.png)

Reply:
we do not want this `{% blocktrans %}` to show in the admin, that's an internal template thing since we manage translations in the code, not in the content/database